Documentation NFTS pallet (#4411)
w3f/polkadot-wiki: docs/learn/learn-nft-pallets.md

  item or the collection)
- Setting or re-setting the price of an item.
- Clearing attributes and metadata of a collection or an item.
- Changing the owner of an item or a collection.
- Transferring an item, as well as creating and canceling transfer approvals of a specific item, or
  an [atomic swap](https://github.com/paritytech/substrate/pull/12285).
- Transferring ownership of an item.

:::info

Metadata, attributes, and settings of an item or collection can be locked. The user can decide what
to lock. Also, unauthorized and/or unprivileged transfers can be prevented by locking mechanism
(unprivileged actions can be re-allowed anytime).

:::

Other features are less common and include:

- Delegating accounts: delegated accounts can approve changes to an item's attributes and transfer
  an item. The item owner always has control and can decide to cancel approvals from a delegated
  account.
- One can also execute pending atomic swaps created by a counterpart.

### Work in Progress

[NFTs fractionalization](https://github.com/paritytech/substrate/pull/12565) will allow the user to:

> - Take ownership of an NFT from the [pallet-uniques](#uniques-pallet)

Reviewer: Should be: [pallet-nfts](#nfts-pallet)

Reviewer 2: Updated! For the next PR, we will send it for your review before merging into the master.

---
id: learn-nft-pallets
title: NFT Pallets
sidebar_label: NFT Pallets
description: An overview of the functionalities of NFT Pallets in Polkadot Ecosystem.
keywords: [NFT, non-fungible token, NFT 2.0, nfts, NFT pallets]
slug: ../learn-nft-pallets
---

> NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
> developers to easily implement NFT-related actions within their dApp.

Reviewer 2: ```suggestion
NFT [pallets](learn-extrinsics.md#pallets-and-extrinsics) allow
developers to easily implement NFT-related actions within their dApp.
```

> ## NFTS Pallet

Reviewer 2: ```suggestion
## NFTs Pallet
```

### Current Implementations

> Nfts is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
> on Westmint that provides a multitude of functions to interact with NFTs.

Reviewer 2: ```suggestion
NFTs is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
on Westmint which provides a multitude of functions to interact with NFTs.
```

> The pallet comes with a new way to configure it, as well as configure collections and items.
> Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow to disable
> functionalities that are not needed in the runtime.

Reviewer 2: By "it", do you mean "NFTs"?

```suggestion
The pallet comes with a new way to configure NFTs, as well as configure collections and items.
Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow disabling
functionalities that are not needed in the runtime.
```

> There are features that common to the nft world such as:

Reviewer 2: ```suggestion
Some features are common to the NFT world, such as:
```

> - Buying an item up for sale.
> - Creating (i.e. mint) and burn (i.e. destroy) a collection of items or a single item (burning must
>   be signed either by the admin of the collection or the owner). The creation of an item usually
>   involves setting some attributes specific to that item (an this is actually what drives the value
>   of the item). The creation of a collection also involves the specification of the maximum number
>   of items the collection has (an this also will drive the value of the collection's items in the
>   future).

Reviewer 2: Not all NFTs are meant for value creation. For instance, Soul bound NFTs, POAPs. 

```suggestion
- Buying an item up for sale.
- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning must
  be signed either by the admin of the collection or the owner). Creating an item usually
    involves setting some attributes specific to that item. Creating a collection also requires the specification of the maximum number of items.
```
